src/pages/library/DataBrowserDetails.js

```diff
@@ -181,7 +183,11 @@ const DataBrowserDetails = ({ id }) => {
             icon('angle-left', { size: 35 })
           ]),
           MainContent(snapshot),
-          Sidebar(snapshot)
+          Sidebar(snapshot, setShowRequestAccessModal),
```

Comment:
Once this has been converted to a component, will these be props of the object argument, rather than positional parameters?

Reply:
What do you mean? Sorry